src/utils/registration-helpers.ts

```diff
@@ -769,31 +777,46 @@ export const registrationLanguageOptions = [
   getLanguageByIso6393Code('mis'),
 ];
 
+const isRegistration = (reg: Registration | RegistrationSearchItem): reg is Registration => {
+  return reg.type === 'Publication';
+};
+
 export const registrationsHaveSamePublicationYear = (
   reg1: Registration | RegistrationSearchItem,
   reg2: Registration | RegistrationSearchItem
```

Comment:
Ser ut som denne alltid tar inn en full Publication og ett søketreff, så egentlig ingen grunn til å håndtere alle kombinasjoner her
```suggestion
  registration: Registration,
  registrationSearchItem: RegistrationSearchItem
```